src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/iFeel_joint_retargeting/robotControl.ini

```diff
@@ -0,0 +1,36 @@
+robot                   ergocub
+
+
+joints_list             ("neck_pitch", "neck_roll", "neck_yaw",
+                         "torso_pitch", "torso_roll", "torso_yaw",
+                         "l_shoulder_pitch", "l_shoulder_roll", "l_shoulder_yaw", "l_elbow", "l_wrist_roll", "l_wrist_pitch", "l_wrist_yaw",
+                         "r_shoulder_pitch", "r_shoulder_roll", "r_shoulder_yaw", "r_elbow", "r_wrist_roll", "r_wrist_pitch", "r_wrist_yaw",
+                         "l_hip_pitch", "l_hip_roll", "l_hip_yaw", "l_knee", "l_ankle_pitch", "l_ankle_roll",
+                         "r_hip_pitch", "r_hip_roll", "r_hip_yaw", "r_knee", "r_ankle_pitch", "r_ankle_roll")
+
+remote_control_boards   ("head", "torso", "left_arm", "right_arm", "left_leg", "right_leg")
+
+# filters
+# if use_*_filter is equal to 0 the low pass filters are not used
+use_joint_velocity_filter          0
+joint_velocity_cut_frequency       10.0
+
+use_wrench_filter                  0
+wrench_cut_frequency               10.0
+
+
+# if true the joint is in stiff mode if false the joint is in compliant mode
+joint_is_stiff_mode     (true, true, true,
+                         true, true, true,
+                         true, true, true, true, true, true, true,
+                         true, true, true, true, true, true, true,
+                         true, true, true, true, true, true,
+                         true, true, true, true, true, true)
+
+# if true a good joint traking is considered mandatory
+good_tracking_required  (false, false, false
+                         true, true, true,
+                         false, false, true, true, false, false, false,
```

Comment:
Similar to above, do we want the last test config be the default ?

Reply:
Having ` false, false, true, true, false, false, false,` means that the tracking for the shoulder roll pitch and the wrist is not checked